src/index.ts

```diff
@@ -2,6 +2,13 @@ import { multinetAxiosInstance, MultinetAxiosInstance } from './axios';
 
 import axios, { AxiosRequestConfig } from 'axios';
 
+export interface Paginated<T> {
+  count: number,
+  next: string,
+  previous: string,
```

Comment:
I believe these can also be `null` if there is no next or previous page.

Reply:
That's right!

See https://github.com/multinet-app/multinetjs/pull/42/commits/b3bc6b705d5dafbf44320c6a32e7bf139c4b7378#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80L7-R8